packages/js-moi-providers/src/base-provider.ts

```diff
@@ -358,6 +358,124 @@ export class BaseProvider extends AbstractProvider {
         }
     }
 
+    /**
+     * Initializes a filter for retrieving newly detected terreracts. 
+     * The filter setup triggers a 1-minute timeout period, and with each subsequent query,
+     * the timeout is reset to 1 minute.
+     * 
+     * @returns {Promise<string[]>} An object containing the filter id for the NewTesseractFilter.
+     * @throws {Error} Throws an error if there is an issue executing the RPC call.
+     */
+    public async getNewTesseractFilter(): Promise<Filter> {
+        try {
+            const params = null;
```

Comment:
unnecessary variable usage can be avoided.

Reply:
6144a3467ccfbaf92c4f59eeb12fffd747006319